schema/src/main/jade-tables/clinical_assertion.table.json

```diff
@@ -157,5 +162,9 @@
       "struct_name": "interpretation_comment",
       "type": "repeated"
     }
-  ]
+  ],
+  "partitioning": {
+    "mode": "date_from_column",
+    "column": "release_date"
+  }
```

Comment:
@quazi-broad like this

Reply:
Thanks for highlighting that! It's more straightforward than I was expecting.